packages/core/src/external/fhir/patient/index.ts

```diff
@@ -61,23 +58,15 @@ export const toFHIR = (patient: Pick<Patient, "id" | "data">): FHIRPatient => {
         };
       }) || [],
   };
-};
+}
 
-export const convertPIToIdentifier = (personalIdentifiers: PersonalIdentifier[]): Identifier[] => {
-  return personalIdentifiers.map(identifier => {
-    if (identifier.type === "driversLicense") {
-      return {
-        system: driversLicenseURIs[identifier.state],
-        value: identifier.value,
-      };
-    } else {
-      return {
-        system: identifierSytemByType[identifier.type],
-        value: identifier.value,
-      };
-    }
+export function getFhirStrongIdsFromPatient(patient: PatientIdAndData): StrongId[] {
+  return (patient.data.personalIdentifiers ?? []).map(id => {
+    if (id.type === "driversLicense")
+      return { value: id.value, system: driversLicenseURIs[id.state] };
```

Comment:
nit: when multi-line `if`, its a good idea to have open/close brackets to avoid something injected there w/ wrong indentation and being interpreted as part of the conditional.